src/ui/app/observable.js

```diff
@@ -191,16 +191,24 @@ export const updateRepairIntensityResult = updateRepairIntensitySubject.map(repa
   }).promise());
 }).share();
 
+export const repairRunResult = repairRunSubject.map(repair => {
+  console.info("Getting Repair Info for: " + repair.clusterName);
+  const params = $.param(repair);
+  return Rx.Observable.fromPromise($.ajax({
+    url: `${URL_PREFIX}/repair_run?${params}`,
+    method: 'GET'
+  }).promise());
+}).share();
 
 export const repairs = Rx.Observable.merge(
-    Rx.Observable.timer(0, POLLING_INTERVAL).map(t => Rx.Observable.just({})),
+    repairRunResult,
     addRepairResult,
     deleteRepairResult,
     updateRepairStatusResult,
     updateRepairIntensityResult
   ).map(s =>
     s.flatMap(t => Rx.Observable.fromPromise($.ajax({
-        url: `${URL_PREFIX}/repair_run`
+        url: `${URL_PREFIX}/repair_run`,
```

Comment:
why the `,` ?

Reply:
Well spotted. I forgot to remove the `,`.